src/components/Menu/MenuItem.tsx

```diff
@@ -118,32 +152,70 @@ class MenuItem extends UIComponent<Extendable<MenuItemProps>, MenuItemState> {
     wrapper: { as: 'li' },
   }
 
+  static autoControlledProps = ['submenuOpen']
+
   state = {
     isFromKeyboard: false,
+    submenuOpen: false,
```

Comment:
won't this override the effect of `submenuOpen={true}` prop being provided by the client? Have taken a look on the logic of `AutoControlledComponent` (`getAutoControlledStateValue`) and it seems to be pretty much the case

Reply:
Yep, we should never have initial value for e state that is part of the Autocontrolled prop. This was leftover from the prev PRs merged here. Will remove it. Thanks for the catch!